src/main/java/picard/sam/HitsForInsert.java

```diff
@@ -235,42 +239,81 @@ public void coordinateByHitIndex() {
         }
     }
 
+    /**
+     * This method lines up alignments in firstOfPairOrFragment and secondOfPair lists so that the first and the second
+     * records of each pair have the same index.
+     */
+    void coordinateByMate() {
+        final List<SAMRecord> newFirstOfPairOrFragment = new ArrayList<>();
+        final List<SAMRecord> newSecondOfPair = new ArrayList<>();
+
```

Comment:
Looks like this file preferred ++i to i++.

Reply:
As you might see ([here](https://github.com/broadinstitute/picard/pull/932/files#diff-fb83641c8b09326f5028047ee001cda6R153) for instance), there was a usage of a postfix increment in this file before. So I just followed this example.
All increment operators in this file changed to prefix form.